msmbuilder/featurizer/featurizer.py

```diff
@@ -629,44 +682,66 @@ def partial_transform(self, traj):
             x.append(result)
         return np.hstack(x)
 
+
+
     def describe_features(self, traj):
-        """Return a list of dictionaries describing the alpha dihedral angle features."""
-        x = []
+        """Return a list of dictionaries describing the dihderal features.
+
+        Parameters
+        ----------
+        traj : mdtraj.Trajectory
+            The trajectory to describe
+
+        Returns
+        -------
+        feature_descs : list of dict
+            Dictionary describing each feature with the following information
+            about the atoms participating in each dihedral
+                - resnames: unique names of residues
+                - atominds: the four atom indicies
+                - resseqs: unique residue sequence ids (not necessarily
+                  0-indexed)
+                - resids: unique residue ids (0-indexed)
+                - featurizer: Alpha Angle
+                - featuregroup: the type of dihedral angle and whether sin or
+                  cos has been applied.
+        """
+        feature_descs = []
         # fill in the atom indices using just the first frame
         self.partial_transform(traj[0])
+        top = traj.topology
         if self.atom_indices is not None:
```

Comment:
This `if` block is somewhat annoying stylistically. It means the whole body of the function has to be indented.

How about
```python
if self.atom_indices is None:
  raise ValueError()
# do the body of the function here
# no need for an else!
```